frontend/vite-project/src/stores/shippingStore.ts

```diff
@@ -0,0 +1,28 @@
+import { create } from 'zustand';
+import { ShippingFormTypes } from '../types/shippingFormTypes'
+
+interface ShippingState {
+  formData: ShippingFormTypes;
+  setFormData: (data: ShippingFormTypes) => void;
+  updateFormField: <K extends keyof ShippingFormTypes>(key: K, value: ShippingFormTypes[K]) => void;
+}
```

Comment:
```suggestion
interface ShippingState {
  formData: ShippingFormTypes;
  actions: ShippingActions
}
```
Combine the Actions type with the ShippingState type.